File: CS/ClientServer_Wcf_StandardAuth.Module/Module.cs
using System;
using System.Collections.Generic;

using DevExpress.ExpressApp;
using System.Reflection;
using DevExpress.ExpressApp.Security;
using DevExpress.Persistent.BaseImpl;
using DevExpress.ExpressApp.Security.Strategy;


namespace ClientServer_Wcf_StandardAuth.Module {
    public sealed partial class ClientServer_Wcf_StandardAuthModule : ModuleBase {
        public ClientServer_Wcf_StandardAuthModule() {
            InitializeComponent();
        }
        protected override IEnumerable<Type> GetDeclaredExportedTypes() {
            List<Type> result = new List<Type>(base.GetDeclaredExportedTypes());
            result.AddRange(new Type[] { typeof(SecuritySystemUser), typeof(SecuritySystemRole), typeof(Person) });
            return result;
        }
    }
}

File: VB/ClientServer_Wcf_StandardAuth.Module/Module.vb
Imports DevExpress.ExpressApp
Imports System.Reflection
Imports DevExpress.ExpressApp.Security
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.ExpressApp.Security.Strategy


Namespace ClientServer_Wcf_StandardAuth.Module
    Public NotInheritable Partial Class ClientServer_Wcf_StandardAuthModule
        Inherits ModuleBase

        Public Sub New()
            InitializeComponent()
        End Sub
        Protected Overrides Function GetDeclaredExportedTypes() As IEnumerable(Of Type)
            Dim result As New List(Of Type)(MyBase.GetDeclaredExportedTypes())
            result.AddRange(New Type() { GetType(SecuritySystemUser), GetType(SecuritySystemRole), GetType(Person) })
            Return result
        End Function
    End Class
End Namespace
